packages/editor/src/components/inner-blocks/style.scss

```diff
@@ -7,3 +7,7 @@
 	left: 0;
 	z-index: z-index(".editor-inner-blocks__small-screen-overlay:after");
 }
+
+.editor-inner-blocks {
+	padding: 10px;
```

Comment:
May want to swap this out with one of our variables to keep it consistent. Perhaps [`$block-padding`](https://github.com/WordPress/gutenberg/blob/286317c8e61b1a2922243875fe1f98c868d7859b/assets/stylesheets/_variables.scss#L55) would work here.

Reply:
Thanks! I didn't realize we had that. It's added now.